apps/api/v1/test/lib/attendees/_post.test.ts

```diff
@@ -0,0 +1,115 @@
+import prismaMock from "../../../../../../tests/libs/__mocks__/prismaMock";
+
+import type { Request, Response } from "express";
+import type { NextApiRequest, NextApiResponse } from "next";
+import { createMocks } from "node-mocks-http";
+import { describe, expect, test } from "vitest";
+
+import handler from "../../../pages/api/attendees/_post";
+
+type CustomNextApiRequest = NextApiRequest & Request;
+type CustomNextApiResponse = NextApiResponse & Response;
+
+describe("POST /api/attendees", () => {
+  describe("Errors", () => {
+    test("Returns 403 if user is not admin and has no booking", async () => {
+      const { req, res } = createMocks<CustomNextApiRequest, CustomNextApiResponse>({
+        method: "POST",
+        body: {
+          bookingId: 1,
+          email: "test@example.com",
+          name: "Test User",
+          timeZone: "UTC",
+        },
+      });
+
+      prismaMock.booking.findFirst.mockResolvedValue(null);
+
+      req.userId = 123;
+      req.isAdmin = false;
```

Comment:
I think we can remove this and do not need to specify that it is false

Reply:
That is correct, adjusted and still working, isAdmin as undefined will work as expected too